src/ripple/app/misc/SHAMapStoreImp.cpp

```diff
@@ -619,11 +619,12 @@ SHAMapStoreImp::health()
 
     NetworkOPs::OperatingMode mode = netOPs_->getOperatingMode();
 
-    std::int32_t age = ledgerMaster_->getValidatedLedgerAge();
-    if (mode != NetworkOPs::omFULL || age >= setup_.ageThreshold)
+    auto age = ledgerMaster_->getValidatedLedgerAge();
+    if (mode != NetworkOPs::omFULL || age.count() >= setup_.ageThreshold)
```

Comment:
I attempted it, but (and this was a judgement call) it seemed to make more of a mess than was worth it.  This variable gets run through `get_if_exists` here:  https://github.com/HowardHinnant/rippled/blob/clocks/src/ripple/basics/BasicConfig.h#L343
and due to the lack of I/O for chrono types fails at that point.  However that can be worked around without too much trouble.  It means making two of the three uses messier:
https://github.com/HowardHinnant/rippled/blob/clocks/src/ripple/app/misc/SHAMapStoreImp.cpp#L627
https://github.com/HowardHinnant/rippled/blob/clocks/src/ripple/app/misc/SHAMapStoreImp.cpp#L698
while making one use cleaner:
https://github.com/HowardHinnant/rippled/blob/clocks/src/ripple/app/misc/SHAMapStoreImp.cpp#L623

This one earned a yawner from me. ;-)  I can go either way.